include/zephyr/shell/shell.h

```diff
@@ -1188,6 +1188,18 @@ int shell_insert_mode_set(const struct shell *shell, bool val);
  */
 int shell_use_colors_set(const struct shell *shell, bool val);
 
+/**
+ * @brief Allow application to control whether terminal is using vt100 commands.
+ * Value is modified atomically and the previous value is returned.
+ *
+ * @param[in] shell	Pointer to the shell instance.
+ * @param[in] val	vt100 mode.
+ *
+ * @retval 0 or 1: previous value
+ * @retval -EINVAL if shell is NULL.
+ */
+int shell_use_vt100_set(const struct shell *shell, bool val);
```

Comment:
It won't fly. In the new coding standard you have to type it:
`int shell_use_vt100_set(const struct shell *sh, bool val);`

Reply:
thank's a lot. Warn message was not so clear to me.